features/product/managing_products/browsing_products.feature

```diff
@@ -0,0 +1,17 @@
+@managing_products
+Feature: Browsing products
+    In order to manage my shop merchandise
+    As an Administrator
+    I want to be able to browse products
+
+    Background:
+        Given the store is available in "English (United States)"
+        And the store has a base currency "Dollar"
+        And the store has "Dice Brewing" and "Eclipse" products
+        And I am logged in as an administrator
+
+    @ui
+    Scenario: Browsing defined products
+        When I want to browse products
```

Comment:
`When I want to browse products` seems to be a little bit more suitable here, as it stresses my intension. I find `I browse products` more suitable for _Given_ step.